frontend/src/screens/OrderListScreen.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { listOrders } from '../actions/orderActions';
+import LoadingBox from '../components/LoadingBox';
+import MessageBox from '../components/MessageBox';
+
+export default function OrderListScreen(props) {
+  const orderList = useSelector((state) => state.orderList);
+  const { loading, error, orders } = orderList;
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(listOrders());
+  }, [dispatch]);
+  const deleteHandler = (order) => {
+    // TODO: delete handler
+  };
+  return (
+    <div>
+      <h1>Orders</h1>
+      {loading ? (
+        <LoadingBox></LoadingBox>
+      ) : error ? (
+        <MessageBox variant="danger">{error}</MessageBox>
+      ) : (
+        <table className="table">
+          <thead>
+            <tr>
+              <th>ID</th>
+              <th>USER</th>
+              <th>DATE</th>
+              <th>TOTAL</th>
+              <th>PAID</th>
+              <th>DELIVERED</th>
+              <th>ACTIONS</th>
+            </tr>
+          </thead>
+          <tbody>
+            {orders.map((order) => (
+              <tr key={order._id}>
+                <td>{order._id}</td>
+                <td>{order.user.name}</td>
```

Comment:
![screenshot png](https://user-images.githubusercontent.com/72513295/127849221-beb4c0e3-9c04-4eb9-adbb-583f372c1884.png)

Reply:
i am stuck here . it can't show user name . how to solve this.

Reply:
you can replace {order.user.name} by {order.shippingAddress.fullname}

Reply:
check your DB and delete orders came from deleted users